packages/amplication-server/src/core/assistant/assistant.service.ts

```diff
@@ -490,7 +501,7 @@ export class AssistantService {
           data: {
             displayName: args.name,
             pluralDisplayName: pluralDisplayName,
-            name: camelCase(args.name),
+            name: pascalCase(args.name),
```

Comment:
would it make sense to move this string manipulation inside the entityService to avoid inconsistencies due to other calls to the same `.entityService.createOneEntity` methods?